packages/core/src/codewhisperer/service/transformByQ/transformationResultsViewProvider.ts

```diff
@@ -362,20 +429,19 @@ export class ProposedTransformationExplorer {
                 getLogger().error(`CodeTransformation: ExportResultArchive error = ${downloadErrorMessage}`)
                 throw new Error('Error downloading diff')
             } finally {
-                cwStreamingClient.destroy()
+                // cwStreamingClient.destroy()
             }
 
             let deserializeErrorMessage = undefined
-            let pathContainingArchive = ''
             try {
                 // Download and deserialize the zip
-                pathContainingArchive = path.dirname(pathToArchive)
-                const zip = new AdmZip(pathToArchive)
-                zip.extractAllTo(pathContainingArchive)
-                diffModel.parseDiff(
-                    path.join(pathContainingArchive, ExportResultArchiveStructure.PathToDiffPatch),
-                    transformByQState.getProjectPath()
-                )
+                // pathContainingArchive = path.dirname(pathToArchive)
+                // const zip = new AdmZip(pathToArchive)
+                // zip.extractAllTo(pathContainingArchive)
+
+                getLogger().info('descriptions', patchFilesDescriptions.toString())
+                diffModel.parseDiff(patchFiles[0], transformByQState.getProjectPath(), patchFilesDescriptions[0])
```

Comment:
Probably add a comment saying multiple patches are returned, once the ZIP is downloaded show the first one to start